jupyterhub_moss/form/option_form.css

```diff
@@ -145,16 +151,17 @@
 
 .environment-add-div {
   display: flex;
-  align-items: flex-start;
+  flex-flow: row wrap;
+  justify-content: flex-start;
```

Comment:
Good point, I like it. Having one field per line will be clearer.

Reply:
Maybe we could also replace the `+` with a more explicit `Add custom environment`

Reply:
Yes, it looks good this way! And +1 for a more explicit button text.

I'm thinking it might be worth putting this part of the form in a foldable details section or something like this, but not for this PR.

Reply:
Done in https://github.com/silx-kit/jupyterhub_moss/pull/99/commits/62ccb2b017cc2bd3dfe04db899ba52a5526a405d